Added props eventOverlapping

rahul-rayo opened the pull request:
- Added props eventOverlapping for events Side by Side instead of overlapping in day and week view.
 
 before
![Screenshot 2024-09-25 at 11 10 28 AM](https://github.com/user-attachments/assets/446a8f44-039d-4648-a6bd-aeea5139f696)

after
![Screenshot 2024-09-25 at 11 10 41 AM](https://github.com/user-attachments/assets/6effdac0-a408-4d4d-b11b-e0d3745f59d9)

rahul-rayo committed:
change colour for check

rahul-rayo committed:
revert change

rahul-rayo committed:
Added excludeSunday props

- Added exclude Sunday props for if the user wants to remove Sunday in the week and month view.

rahul-rayo committed:
fix issue

rahul-rayo committed:
Update datetime.ts

 - Remove console

rahul-rayo committed:
Update App.tsx

rahul-rayo committed:
Merge branch 'acro5piano:main' into main

rahul-rayo committed:
Added props eventOverlapping

 - Added props eventOverlapping for events Side by Side instead of overlapping in day and week view.

rahul-rayo committed:
Merge branch 'event-overlapping'

acro5piano commented:
@rahul-rayo Thanks! I'll take a look.

acro5piano commented:
This is a breaking change so I'll bump a major version.

acro5piano commented:
@rahul-rayo 

Thank you so much for your pull request. I've took a look at the deployed preview and found the events are still overwrapping, although the option is enabled by default - can you provide an example in the storybook for visibility?

![image](https://github.com/user-attachments/assets/b9dfa473-0b7c-41b4-898a-ee4fa552ea7e)

rahul-rayo commented:
Ok @acro5piano if i get time will do.

acro5piano commented:
@rahul-rayo Thank you!

rahul-rayo committed:
Customised day-view

- Customised day-view in added week days in header.

rahul-rayo committed:
Update datetime.ts

rahul-rayo committed:
Changes in day view

 - Day view in changed like if i click to particular date then automatic change data.

rahul-rayo committed:
Update CalendarBody.tsx

rahul-rayo committed:
Today date highlighted

arisyo13 commented:
Any plan to release this PR soon?

acro5piano commented:
@arisyo13 Thanks for flagging this. If you create a complete PR forking from this one, it's welcome. 
The biggest blocker is that this PR will breaks the default - which means major version bump.

rahul-rayo committed:
Update HourGuideColumn.tsx

nizamozcan commented:
> @arisyo13 Thanks for flagging this. If you create a complete PR forking from this one, it's welcome. The biggest blocker is that this PR will breaks the default - which means major version bump.

Hello,
Is there an update for this Pr?
I am having trouble with conflicting appointments

acro5piano commented:
@nizamozcan @Sheena1234 
I think @rahul-rayo 's approach is correct, but the default is inverted - I will look into it when I have time, but if you can work on it please update rahul's PR.

NicolasJott commented:
This would be super helpful
